lms/templates/student_account/enterprise_sidebar.html

```diff
@@ -0,0 +1,13 @@
+<div id="enterprise-content-container" class="enterprise-content">
+    <div class="centered-div">
+        % if enterprise_logo_url:
+            <img src="${enterprise_logo_url}" alt="" class="enterprise-logo">
+        % endif
+        <h2>
+            ${enterprise_branded_welcome_string}
+        </h2>
+        <h2>
+            ${platform_welcome_string}
+        </h2>
+    </div>
+</div>
```

Comment:
So, this raises an interesting question. If we embed the template, etc, in edx-enterprise, then we're tied to having edx-enterprise be installed in edx-platform, rather than something we can eventually spin out into an IDA. This is a discussion that's been ongoing for quite a while, and one we don't really have an answer to yet.